src/coreclr/jit/emitxarch.cpp

```diff
@@ -14681,18 +14698,6 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
             result.insThroughput = PERFSCORE_THROUGHPUT_25C;
             break;
 
-        case INS_mov_xmm2i:
```

Comment:
It looks like these perfscores and the ones used for `INS_movd` are tracked slightly differently?